explorer/nodewatch/openapi/openapi.yaml

```diff
@@ -0,0 +1,436 @@
+openapi: 3.0.3
+info:
+  title: NodeWatch API Documentation
+  description: >
+    API providing information about NEM and Symbol blockchain networks, including nodes, network heights, and time-series node statistics.
+  version: 1.0.0
+
+servers:
+  - url: https://nodewatch.symbol.tools/testnet
+    description: Testnet server
+  - url: https://nodewatch.symbol.tools
+    description: Mainnet server
+
+tags:
+  - name: NEM nodes
+    description: Endpoints providing NEM node and height information.
+  - name: Symbol nodes
+    description: Endpoints providing Symbol node, height, and statistics.
+
+paths:
```

Comment:
There are 2 missing endpoints:
`/api/nem/chart/height`
`/api/symbol/chart/height`

These 2 endpoints are used for the nodewatch page.
unsure should this be added to this

Reply:
I intentionally left out these endpoints, as they don’t seem particularly useful outside of Nodewatch.